package.yaml

```diff
@@ -34,6 +34,7 @@ dependencies:
 - filepath
 - directory
 - cassava
+- tree-view
```

Comment:
https://hackage.haskell.org/package/tree-view

This library is simple and useful enough. Barely any deps too:

```
base (<5), containers, mtl
```

It also supports collapsible HTML output.

Reply:
Oh awesome.  I did not want to write that. heh